builtin/game/features.lua

```diff
@@ -42,8 +42,9 @@ core.features = {
 	override_item_remove_fields = true,
 	hotbar_hud_element = true,
 	bulk_lbms = true,
-	abm_without_neighbors = true,
-	biome_weights = true,
+  abm_without_neighbors = true,
```

Comment:
Indentation here is wrong. I noticed that this tends to happen when using GitHub's online editor (maybe it doesn't see `.editorconfig` due to some kind of sandboxing?).